frontend/src/components/Profile.vue

```diff
@@ -0,0 +1,184 @@
+            </div>
+          </div>
+        </div>
+
+        <div class="container__radio">
+          <label for="usertype" class="radio-label">Status*</label>
+          <div class="container__radio-item1">
+            <input type="radio" name="status" id="newcomer" checked />
+            <label for="newcomer">Newcomer</label>
+            <span class="container__radio-check"></span>
+          </div>
+
+          <div class="container__radio-item1">
+            <input type="radio" name="status" id="volunteer" />
+            <label for="volunteer">Volunteer</label>
+            <span class="container__radio-check"></span>
+          </div>
+        </div>
+
+        <div class="container__group">
+          <input
+            type="email"
+            placeholder="E-mail*"
+            name="email"
+            id="email"
+            required
+          />
+        </div>
+        <div class="container__group">
+          <input
+            type="password"
+            placeholder="Password*"
+            name="password"
+            id="password"
+            required
+          />
+        </div>
+
+        <div class="container__group">
+          <div class="container__select">
+            <select name="Gender" id="gender" required>
+              <option value="Gender" disabled selected>Gender*</option>
+              <option value="Male">Male</option>
+              <option value="Female">Female</option>
+              <option value="Other">Other</option>
+            </select>
+            <span class="container__select-icon"
+              ><i class="zmdi zmdi-chevron-down"></i
+            ></span>
+          </div>
+        </div>
+
+        <div class="container__group">
+          <input
+            type="text"
+            placeholder="Nationality"
+            name="nationality"
+            id="nationality"
+          />
+        </div>
+
+        <div class="container__group">
+          <div class="container__select">
+            <select name="languages" id="languages">
+              <option value="Language" disabled selected>Languages</option>
+              <option value="French">English</option>
+              <option value="Dutch">Dutch</option>
+              <option value="English">French</option>
+              <option value="Russian">Russian</option>
+              <option value="Spanish">Spanish</option>
+              <option value="Arabic">Arabic</option>
+              <option value="Hindu">Hindu</option>
+              <option value="Chinese">Chinese</option>
+              <option value="Italian">Italian</option>
+            </select>
+            <span class="container__select-icon"
+              ><i class="zmdi zmdi-chevron-down"></i
+            ></span>
+          </div>
+        </div>
+        <div class="container__group">
+          <div class="container__select">
+            <select name="skills" id="skills">
+              <option value="Skills" disabled selected>Skills</option>
```

Comment:
same here

Reply:
hm not sure, why can not users change/add skills?

Reply:
They should be able to! But only after pressing the edit button.

Reply:
oh, okay..or we can have just 1 button to save..and they can edit by default..what do you think?

Reply:
either way is fine

Reply:
well I've never seen that when you look at your profile you can edit by default, and anyway when you do, you don't see select buttons 🙂